src/Bids/Lebensohl.hs

```diff
@@ -278,6 +284,38 @@ b1NoBM2N3C3N :: Action
 b1NoBM2N3C3N = bid3N_ [T.Hearts, T.Spades] True
 
 
+-- Stayman-like cue bids
+cueBid_ :: T.Suit -> Bool -> Action
+cueBid_ oppsSuit shouldHaveStopper = do
+    NT.gameForcing
```

Comment:
Consider also requiring that you're balanced (or at least semibalanced?) to bid this: ending up in notrump with a void can be awkward. Also consider requiring that you have at most 4 cards in the opponent's suit. That's not important when the opponents make a two-suited bid, but I had an example where RHO overcalled a single-suited spade, and we were 5404. Stayman would probably be the right choice if RHO was two-suited (surely LHO will run to the second suit, which will be in our void), but RHO was single-suited, and I think a double is the right call.